scripts/js/upgrade/index.js

```diff
@@ -0,0 +1,330 @@
+const { ApiPromise, WsProvider, Keyring } = require('@polkadot/api');
+const { blake2AsHex } = require('@polkadot/util-crypto');
+const fs = require('fs')
+const util = require('util');
+const exec = util.promisify(require('child_process').exec);
+
+const run = async () => {
+  let exitCode = 0;
+  try {
+    console.log("Parsing Args ...")
+    // 0 & 1 are command context
+    const endpoint = "ws://0.0.0.0:9946";
+    const endpointRelay = "ws://0.0.0.0:9944";
+    const seeds = ["//Alice", "//Bob", "//Charlie"];
+    const wasmFile = process.argv[2];
+    const targetDockerTag = process.argv[3];
+
+    const wsProvider = new WsProvider(endpoint);
+    const wsProviderRelay = new WsProvider(endpointRelay);
```

Comment:
I would only instantiate these closer to their first usage but leave that up to you.